### PR TITLE
Update communication.md

### LINKED ISSUE #17: communication (Fixed the issue)
Error message in communication.md MD013/ line-length: line length [Expected: 80; Actual: 107]

### PR DESCRIPTION
**Pull request by ermishina**
---
name: communication
about: A template PR for code review with a checklist
---

### Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [ ] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [ ] The function has correct type annotations

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

### CODE REVIEW

File: collaboration/communication.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+# Communication
+
+______________________________________________________________________
+
+## Communication Schedule
+
+| Day           | How             | The topic of discussion              |
+|---------------|-----------------|-----------------------------------------------|
+| Fri, 12/20, 3pm ETZ| Zoom Meeting    | Initial Briefing: Team Setup and Objectives|
+| Sun, 12/22, 5pm ETZ| Zoom Meeting    | Review, Formatting, and Completion of Tools of Collaborative Development and Defining Group Norms|
```
Comment (Azizsin7) [inline collaboration/communication.md]: check the line length

File: collaboration/communication.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+| Sun, 12/29 | Zoom Meeting  | Repository Review, Task Allocation, General Q&A, Goal Setting, and Project Kickoff|
```
Comment (Azizsin7) [inline collaboration/communication.md]: check the line length

File: collaboration/communication.md [hunk 3]
```diff
@@ -1 +1,66 @@
...
+| Mon, 12/30 | Zoom Meeting    | Issue Discussion, Problem-Solving, and Goal Setting for the Next Task|
```
Comment (Azizsin7) [inline collaboration/communication.md]: check the line length

File: collaboration/communication.md [hunk 4]
```diff
@@ -1 +1,66 @@
...
+- **Slack**:Always
+- **Zoom Meetings**:Weekly/ 2 times per week/ 3h
+
+______________________________________________________________________
+
+## Availability
+
+### Availability for calling/messaging
+
+| Name          | Monday      | Tuesday     | Wednesday  | Thursday   | Friday     | Saturday   | Sunday     |
```
Comment (Azizsin7) [inline collaboration/communication.md]: check the line length

File: collaboration/communication.md [hunk 5]
```diff
@@ -1 +1,66 @@
...
+|---------------|-------------|-------------|------------|------------|------------|------------|------------|
+| Aziz Azizi    | 10-12pm ETZ  |10-12pm ETZ |10-12pm ETZ|10-12pm ETZ | 10-12pm ETZ  |10-12pm ETZ  |10-12pm ETZ  |
```
Comment (Azizsin7) [inline collaboration/communication.md]: check the line length

**Review by Azizsin7 — CHANGES_REQUESTED** [review]
check the line length
48d754:1d3b6201ada943a88e3c1b7191afb983
Error: collaboration/communication.md:20:81 MD013/line-length Line length [Expected: 80; Actual: 139]
Error: collaboration/communication.md:21:81 MD013/line-length Line length [Expected: 80; Actual: 114]
Error: collaboration/communication.md:22:81 MD013/line-length Line length [Expected: 80; Actual: 103]
Error: collaboration/communication.md:38:81 MD013/line-length Line length [Expected: 80; Actual: 110]
Error: collaboration/communication.md:40:81 MD013/line-length Line length [Expected: 80; Actual: 113]
Error: collaboration/communication.md:41:81 MD013/line-length Line length [Expected: 80; Actual: 107]
Error: collaboration/communication.md:42:81 MD013/line-length Line length [Expected: 80; Actual: 129]
Error: collaboration/communication.md:43:81 MD013/line-length Line length [Expected: 80; Actual: 112]
Error: collaboration/communication.md:44:81 MD013/line-length Line length [Expected: 80; Actual: 111]
Error: collaboration/communication.md:45:81 MD013/line-length Line length [Expected: 80; Actual: 110]
Error: collaboration/communication.md:49:81 MD013/line-length Line length [Expected: 80; Actual: 95]
Error: collaboration/communication.md:51:81 MD013/line-length Line length [Expected: 80; Actual: 94]
Error: collaboration/communication.md:52:81 MD013/line-length Line length [Expected: 80; Actual: 94]
Error: collaboration/communication.md:54:81 MD013/line-length Line length [Expected: 80; Actual: 88]
Error: collaboration/communication.md:55:81 MD013/line-length Line length [Expected: 80; Actual: 94]
Error: collaboration/communication.md:5[6](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-06/actions/runs/12551767828/job/34996716211#step:4:7):81 MD013/line-length Line length [Expected: 80; Actual: 96]
Error: README.md:2 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]

**Review by Azizsin7 — APPROVED** [review]

**Review by Azizsin7 — APPROVED** [review]

**Review by Azizsin7 — APPROVED** [review]
